Kernel/ACPI/ACPIStaticParser.h

```diff
@@ -61,8 +61,8 @@ class ACPIStaticParser : ACPIParser {
     PhysicalAddress m_rsdp;
     PhysicalAddress m_main_system_description_table;
 
-    OwnPtr<ACPI::MainSystemDescriptionTable> m_main_sdt;
-    OwnPtr<ACPI::FixedACPIData> m_fadt;
+    OwnPtr<ACPI::MainSystemDescriptionTable> m_main_sdt { nullptr };
+    OwnPtr<ACPI::FixedACPIData> m_fadt { nullptr };
```

Comment:
No need to explicitly initialize these. `OwnPtr` automatically sets itself to `nullptr`.